sql-jdbc/src/TableauConnector/opensearch_sql_jdbc/manifest.xml

```diff
@@ -2,7 +2,7 @@
 
 <connector-plugin class='opensearch_jdbc' superclass='jdbc' plugin-version='1.1.0.0' name='OpenSearch' version='18.1' min-version-tableau='2021.1'>
   <vendor-information>
-      <company name="OpenSearch for ES"/>
+      <company name="OpenSearch Project"/>
       <support-link url="https://github.com/opensearch-project/sql"/>
   </vendor-information>
   <connection-customization class="opensearch_jdbc" enabled="true" version="10.0">
```

Comment:
What about Line 9 for "Vendor"?

Reply:
"OpenSearch JDBC doesn't make sense to me as the Vendor. What do other connectors do?

Reply:
I have only dremio example: https://github.com/dremio-hub/tableau-connector/blob/master/manifest.xml